docs/guides/custom-domains/index.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Custom Domains
+---
+
+## Overview
+[myzrok.io](https://myzrok.io) is a hosted zrok-as-a-service offering that provides a way for you bring a custom DNS name for zrok shares.
+For example, let's say you own the domain `foo.example.io`, you can leverage zrok custom domains to
+create ephemeral shares such as: `https://vw8jbg4ijz5g.foo.example.io`
+or [reserved shares](/concepts/sharing-reserved.md) such as `https://myshare.foo.example.io`.
+
+Custom domains require a Pro subscription with [myzrok.io](https://myzrok.io).
+If you don't already have an account, you can sign up for one [here](https://myzrok.io).
+
+[myzrok.io](https://myzrok.io) provides a guided setup with just a few easy steps!
+
+1. Bring your own custom domain name
+2. Create DNS validation records and a CNAME record
```

Comment:
```suggestion
2. Create DNS validation CNAME-record for certificate management and A-record round-robin for the frontend
```

Reply:
The `-record` suffixes might be too noisy, but I thought it best to avoid implying the A records are for "validation," when only the CNAME record is for validation, i.e., delegating cert mgmt authority, and the A records are a round robin for the frontend proxy.

Reply:
round-robin is a load balancer thing, not a DNS thing, so I'd prefer not to muddy the waters with that term here. How about something more like "Create DNS records for certificate validation and traffic routing" (leave it generic and the specifics will get covered below).

Reply:
Ah, I stand corrected and did not know that. My bad